plugins/bcli.c

```diff
@@ -656,7 +656,14 @@ static struct command_result *getchaininfo(struct command *cmd,
                                            const char *buf UNUSED,
                                            const jsmntok_t *toks UNUSED)
 {
-	if (!param(cmd, buf, toks, NULL))
+	/* FIXME(vincenzopalazzo): Inside the JSON request,
+         * we have the current height known from Core Lightning. Therefore,
+         * we can attempt to prevent a crash if the 'getchaininfo' function returns
+         * a lower height than the one we already know, by waiting for a short period.
+         * However, I currently don't have a better idea on how to handle this situation. */
+	if (!param(cmd, buf, toks,
+		   p_opt_any(),
```

Comment:
Hmm, just do `p_req("lsat_height", param_number, &height)`?

Reply:
Yeah I think this p_opt_any is a little bit dangerous, I can use the UNUSED macros to avoid warnings